src/NUnitFramework/TestBuilder.cs

```diff
@@ -11,213 +11,214 @@
 
 #nullable enable
 
-namespace NUnit.Framework.Tests.TestUtilities
+namespace NUnit.Framework.Tests.TestUtilities;
+
+/// <summary>
+/// Utility Class used to build and run NUnit tests used as test data
+/// </summary>
+internal static class TestBuilder
 {
-    /// <summary>
-    /// Utility Class used to build and run NUnit tests used as test data
-    /// </summary>
-    internal static class TestBuilder
+    #region Build Tests
+
+    public static TestSuite MakeSuite(string name)
     {
-        #region Build Tests
+        return new TestSuite(name);
+    }
 
-        public static TestSuite MakeSuite(string name)
-        {
-            return new TestSuite(name);
-        }
+    public static TestSuite MakeFixture(Type type)
+    {
+        return new DefaultSuiteBuilder().BuildFrom(new TypeWrapper(type));
+    }
 
-        public static TestSuite MakeFixture(Type type)
-        {
-            return new DefaultSuiteBuilder().BuildFrom(new TypeWrapper(type));
-        }
+    public static TestSuite MakeFixture(object fixture)
+    {
+        TestSuite suite = MakeFixture(fixture.GetType());
+        suite.Fixture = fixture;
+        return suite;
+    }
 
-        public static TestSuite MakeFixture(object fixture)
-        {
-            TestSuite suite = MakeFixture(fixture.GetType());
-            suite.Fixture = fixture;
-            return suite;
-        }
+    public static TestSuite MakeParameterizedMethodSuite(Type type, string methodName)
+    {
+        var suite = MakeTestFromMethod(type, methodName) as TestSuite;
+        Assert.That(suite, Is.Not.Null, "Unable to create parameterized suite - most likely there is no data provided");
+        return suite!;
+    }
 
-        public static TestSuite MakeParameterizedMethodSuite(Type type, string methodName)
-        {
-            var suite = MakeTestFromMethod(type, methodName) as TestSuite;
-            Assert.That(suite, Is.Not.Null, "Unable to create parameterized suite - most likely there is no data provided");
-            return suite;
-        }
+    public static TestMethod MakeTestCase(Type type, string methodName)
+    {
+        var test = MakeTestFromMethod(type, methodName) as TestMethod;
+        Assert.That(test, Is.Not.Null, $"Unable to create TestMethod from {methodName}");
 
-        public static TestMethod MakeTestCase(Type type, string methodName)
-        {
-            var test = MakeTestFromMethod(type, methodName) as TestMethod;
-            Assert.That(test, Is.Not.Null, $"Unable to create TestMethod from {methodName}");
+        return test!;
```

Comment:
No need for null suppression operator. NUnit.Analyzer does the suppressing.

Reply:
Now then.....